defmt/src/export/mod.rs

```diff
@@ -8,6 +8,9 @@ use crate::{Format, Formatter, Str};
 pub use self::integers::*;
 pub use bitflags::bitflags;
 
+#[cfg(feature = "bitflagsv2")]
+pub use bitflagsv2::bitflags as bitflagsv2;
```

Comment:
may I ask for the `bitflags` traits (`Bits` & `Flags`) to also be re-exported.
I need them, and other users might want them too, for a project and would prefer relying on the version installed by `defmt` instead of listing it manually.
Refers to #923

Reply:
That's reasonable - perhaps for v2 import the whole crate like:

```rust
pub use bitflags::{bitflags, Bits, Flags};

#[cfg(feature = "bitflagsv2")]
pub use bitflagsv2;
```

Reply:
@manushT any thoughts on this one?